### ISSUE TITLE
Access camera

### ISSUE THREAD

**Issue by Fabiyo-90** [Notalib/nativescript-webview-ext#120]
How can I access the camera?
It would have been interesting to include such a feature in the plugin

**Comment by farfromrefug** [comment]
@Fabiyo-90 i have implemented it in `@nativescript-community/ui-webview` you just need to add `webRTC="true"` to your  webview. It is a fork of this component so you can directly replace it.
@m-abs i would like to contact you on discord if possible to talk about maintaining yours and mine simultaneously. Also need to explain why i had to fork

**Comment by m-abs** [comment]
@farfromrefug 
I welcome a fork of this plugin.
We developed it for our own app, which is still on NativeScript 6 and we don't have much time to develop this anymore.

**Comment by farfromrefug** [comment]
@m-abs ok good to know. I ll move along then. Thanks

**Comment by Fabiyo-90** [comment]
Thank you very much, hope this will help me to easily set up video calls via webrtc.
It's pretty sad that I couldn't find a plugin on it.
I went to see the page "https://github.com/nativescript-community/ui-webview" and I must say that this specificity is missing in the documentation, it will help more than one

**Comment by farfromrefug** [comment]
@Fabiyo-90 yes the doc is not up to date yet. as usual the typings are (which you should always base yourself on).
And PRs are welcome. 
if you have another issue please post it there to not spam this repo.
